nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/ManagementBAFTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour AL KOTOB
+ */
+package org.nuxeo.rest.management;
+
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.nuxeo.ecm.core.bulk.io.BulkConstants.STATUS_COMMAND_ID;
+import static org.nuxeo.ecm.core.bulk.io.BulkConstants.STATUS_ENTITY_TYPE;
+import static org.nuxeo.ecm.core.io.registry.MarshallingConstants.ENTITY_FIELD_NAME;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+import javax.inject.Inject;
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.nuxeo.ecm.core.bulk.message.BulkStatus.State;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.TransactionalFeature;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * @since 11.1
+ */
+public abstract class ManagementBAFTest extends ManagementBaseTest {
+
+    private static final String STATE = "state";
```

Comment:
Why not protected or public as the other ones?

Reply:
Arf, IDE refactoring...
One could say static final has no reason not to be public which is usually my go to.
But we could argue this field is not needed in extended classes so hiding it helps to not confuse people ?
I leave it to you !

Reply:
As long as it is consistent: if all the test constants are public then make it public.

Reply:
Did it and did the same for `ManagementBaseTest`